src/utils.js

```diff
@@ -696,3 +696,45 @@ export function deepAccess(obj, path) {
   }
   return obj;
 }
+
+/**
+ * Build an object consisting of only defined parameters to avoid creating an
+ * object with defined keys and undefined values.
+ * @param {object} object The object to pick defined params out of
+ * @param {string[]} params An array of strings representing properties to look for in the object
+ * @returns {object} An object containing all the specified values that are defined
+ */
+export function getDefinedParams(object, params) {
+  return params
+    .filter(param => object[param])
+    .reduce((bid, param) => Object.assign(bid, { [param]: object[param] }), {});
+}
+
+/**
+ * @typedef {Object} MediaTypes
+ * @property {Object} banner banner configuration
+ * @property {Object} native native configuration
+ * @property {Object} video video configuration
+ */
+
+/**
+ * Validates an adunit's `mediaTypes` parameter
+ * @param{MediaTypes} mediaTypes mediaTypes parameter to validate
+ * @returns{boolean} If object is valie
+ */
+export function mediaTypesValid(mediaTypes) {
```

Comment:
For all functions returning boolean, naming should start with `is` or `has`.  I was going through it over here https://google.github.io/styleguide/jsguide.html#naming-method-names. 
So this function can be renamed as `hasValidMediaType` or `isValidMediaType`